client/src/pages/Map.js

```diff
@@ -1,12 +1,13 @@
 import React, { Component } from 'react';
+import OLMap1 from '../components/map/openlayers/OLMap'
 
 
 class Map extends Component {
     render() {
         return (
-            <h1>Map Page</h1>
+            <h1><OLMap1/></h1>
```

Comment:
h1 not need

Reply:
Just missed this single piece of code :)